ark/utils/data_utils.py

```diff
@@ -559,7 +559,7 @@ def download_example_data(save_dir: Union[str, pathlib.Path]):
     # Downloads the dataset
     ds = datasets.load_dataset("angelolab/ark_example")
 
-    data_path = pathlib.Path(ds["base_dataset"]["Data Path"][0]) / "input_data"
```

Comment:
This is the actual path that huggingface produces. Just changing it here will cause an error if it's not changed in the raw files, right?

Reply:
It's changed in the hugging face repo as well [here](https://huggingface.co/datasets/angelolab/ark_example/blob/main/ark_example.py). It was inconsistent with what we have here in the notebooks.